service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/UploadTranslationsSubmittedControllerTest.java

```diff
@@ -237,38 +227,36 @@ void shouldSendOrdersByPostWhenOrderTranslated() {
 
     @Test
     void shouldNotifyRepresentativesServedDigitallyWhenOrderTranslated() {
-        CaseData caseData = caseData();
-        CallbackRequest request = CallbackRequest.builder()
-            .caseDetails(asCaseDetails(caseData))
-            .build();
 
-        postSubmittedEvent(request);
+        postSubmittedEvent(caseData());
 
         checkUntil(() -> verify(notificationClient, timeout(ASYNC_METHOD_CALL_TIMEOUT)).sendEmail(
             eq(ITEM_TRANSLATED_NOTIFICATION_TEMPLATE),
-            eq(REPRESENTATIVE_DIGITAL.getValue().getEmail()), eqJson(NOTIFICATION_PARAMETERS),
+            eq(REPRESENTATIVE_DIGITAL.getValue().getEmail()), eq(NOTIFICATION_PARAMETERS),
             eq(NOTIFICATION_REFERENCE)
         ));
     }
 
     @Test
     void shouldNotifyRepresentativesServedByEmailWhenOrderTranslated() {
-        CaseData caseData = caseData();
-        CallbackRequest request = CallbackRequest.builder()
-            .caseDetails(asCaseDetails(caseData))
-            .build();
 
-        postSubmittedEvent(request);
+        postSubmittedEvent(caseData());
 
         checkUntil(() -> verify(notificationClient, timeout(ASYNC_METHOD_CALL_TIMEOUT)).sendEmail(
```

Comment:
don't think the timeout is required here, should be handled natively by the `checkUntil` method

Reply:
same for other places in this class as well